🌐 Update German translation for `docs/de/docs/features.md`

nilslindemann opened the pull request:
Updates `de/docs/features.md`, which already exists in the main branch.

← `index.md` (#10283)
→ `learn/index.md` (#10497)
→ `python-types.md` (#10287)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

Commit:
German translation of docs/features.md

* Full translation
* Updates already existing de/docs/features.md
* In sync with current en/docs/features.md
* Hand-translated (Google translate rarely used)
* Checked for errors using LanguageTool
* Manually checked

tiangolo commented:
📝 Docs preview for commit 8895b9b20cd60294545118dfc52ffc2f10ae4adb at: https://301eee67.fastapitiangolo.pages.dev

Commit:
Weniger eindeutschen

Die Englischen Worte sind in IT-Kreisen verbreiteter.

Siehe die Diskussion bei #10290

tiangolo commented:
📝 Docs preview for commit af0fa80fa0a7ba71a896c105bd9be5f62ada1be0 at: https://a2344b4b.fastapitiangolo.pages.dev

Commit:
Weniger eindeutschen

Die englischen Worte sind in IT-Kreisen verbreiteter.

Siehe die Diskussion bei #10290

tiangolo commented:
📝 Docs preview for commit 85153e60a3f083896c11a9c047a1d6d0ef05d6c3 at: https://97449033.fastapitiangolo.pages.dev

Commit:
Standardwert -> Defaultwert

Passt besser. Steht auch bei Leo als erste Übersetzung

tiangolo commented:
📝 Docs preview for commit 5e721d608af3564f0d6b41c00b9a0e11644b7adf at: https://1263f7a4.fastapitiangolo.pages.dev

Commit:
Weniger eindeutschen

Die englischen Worte sind in IT-Kreisen verbreiteter.

Siehe die Diskussion bei #10290

tiangolo commented:
📝 Docs preview for commit fe9d130a2f69890dba23fa2fecfe6a5f8e5a5a73 at: https://691a5c8d.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

tiangolo commented:
📝 Docs preview for commit 0957284f6e43431b207e7dada30d63d1506bb5c7 at: https://c616b407.fastapitiangolo.pages.dev

Commit:
Improve translation of "path operation"

Plus two better formulations

tiangolo commented:
📝 Docs preview for commit ae29828c2d3fa2693393c465bf373c197e68d36f at: https://a4654325.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

tiangolo commented:
📝 Docs preview for commit 6355876355b992e2285d4c9b0ef1785dc507aa26 at: https://3c41267a.fastapitiangolo.pages.dev

Commit:
A few improvements

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

Commit:
Merge branch 'de/docs/features.md' of https://github.com/nilslindemann/fastapi into de/docs/features.md

tiangolo commented:
📝 Docs preview for commit 09eb84c65fd7b95e0296a3e0b7ec19d811e84ccf at: https://8a6f038b.fastapitiangolo.pages.dev

Commit:
"Performance" -> "Performanz"

Changed my opinion. https://de.wikipedia.org/wiki/Rechenleistung uses "Performanz", so I use it too.

tiangolo commented:
📝 Docs preview for commit f7bb2fd19a173975434ebabb5b796e25345a1429 at: https://86b25388.fastapitiangolo.pages.dev

Commit:
Remove fork sync conflict

... and use "Standard" instead of "normal" on the way.

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

tiangolo commented:
📝 Docs preview for commit a8888bf90836129460aa5be475deea52e9144aea at: https://da067f3f.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

tiangolo commented:
📝 Docs preview for commit f783bd5373941fd0b7f9f19a58e1ca677f30ea14 at: https://90d62116.fastapitiangolo.pages.dev

Commit:
Sync with #10392

tiangolo commented:
📝 Docs preview for commit fff815f16649b53b2fe1c043f2118140afb6419b at: https://53ca3bed.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

tiangolo commented:
📝 Docs preview for commit a8575e412f5869dda8d550372133e9bd7133443f at: https://baa034c5.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

Commit:
Use typographic quotes

tiangolo commented:
📝 Docs preview for commit aced185ac089fb35390453c8d93375d037a1c438 at: https://31db45ed.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

tiangolo commented:
📝 Docs preview for commit 70d479df43810daf4504c940c638a1b957bb7f14 at: https://311a5d4f.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

tiangolo commented:
📝 Docs preview for commit b90497fc993d8a583d3d882be0abff4b6dfec94b at: https://6d31f4ec.fastapitiangolo.pages.dev

Commit:
"Standard " -> "Standard-"

Commit:
Forgot some "Antwort" -> "Response"

tiangolo commented:
📝 Docs preview for commit 65a4d16a9b8fdb76ddfb0a98871e915fc77d2a1b at: https://d703e281.fastapitiangolo.pages.dev

Commit:
"Python " -> "Python-", etc.

tiangolo commented:
📝 Docs preview for commit 9d6b468079531ef68dee1619798dfd30537729c5 at: https://076e58a0.fastapitiangolo.pages.dev

Commit:
"das gleiche" vs "dasselbe"

tiangolo commented:
📝 Docs preview for commit 3042a615d99e795c6a85173dd5303c77849e1ff3 at: https://2398a889.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

tiangolo commented:
📝 Docs preview for commit a8b9bdc05c2ee424ba4ef7abce44ee6377200ed4 at: https://eab28008.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

Commit:
Remove "-"

tiangolo commented:
📝 Docs preview for commit dbada9dad5662275c02b8493358da9c5f33e1774 at: https://e7e4b5ca.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/features.md

tiangolo commented:
📝 Docs preview for commit d868f236e8e7453bd9d171a0cd7e16ed427c9e65 at: https://cf579443.fastapitiangolo.pages.dev

tiangolo commented:
It seems this was covered in another PR, so I'll close this one now. :nerd_face:

tiangolo committed:
Merge branch 'master' into de/docs/features.md
